src/page_components/listing/ListingDetailHeader.scss

```diff
@@ -71,23 +90,9 @@
 }
 
 .detail-header__subtitle {
-  @apply text-sm;
+  @apply text-tiny;
```

Comment:
This was an a11y issue, so in core the subtitle text will be little bit bigger and a little bit darker